### ISSUE TITLE
Error when running lslint: __init__() got an unexpected keyword argument 'cancel'

### ISSUE THREAD

**Issue by thickbrick** [buildersbrewery/sublime-lsl#2]
## When I run lslint via ctrl+b, I get an error message:

```__init__() got an unexpected keyword argument 'cancel'```

Removing the ```"cancel": { "kill" : true},``` part from LSL.sublime-build allows it to work as expected.

### Environment

```text
=BB= LSL:

    version: 1.2.0
    installed via Package Control: True

Sublime Text:

    channel: stable
    build: 3143
    platform: Linux
    architecture: x64
```

**Comment by buildersbrewery** [comment]
Interesting, this works on Windows. Will try to update by tonight.

**Comment by buildersbrewery** [comment]
fixed in `v1.2.1`
